pkg/tree/node/node.go

```diff
@@ -264,6 +264,12 @@ func (n *Node) Union(source *Node) {
 	for _, child := range source.Children {
 		if !n.HasChild(child.GetName()) {
 			n.AddChild(child)
+		} else if child.GetChildren() != nil {
+			// union at child level
+			n.GetChildS(child.GetNameS()).Union(child)
```

Comment:
can we use byte methods? GetChild instead of GetChildS

Reply:
done.